### ISSUE TITLE
Template fails to import into Zabbix 4.0

### ISSUE THREAD

**Issue by danboid** [rvalitov/zabbix-php-fpm#5]
I am unable to import the current zabbix-php-fpm template into Zabbix 4.0.

On my first attempt I got the error:

```
Import failed

    Invalid tag "/zabbix_export/version": unsupported version number.
```

So I edited the template and changed the version tag to 4.0 then I tried again:

```
Import failed

    Invalid tag "/zabbix_export/templates/template(1)": unexpected tag "tags".
```

So it seems Zabbix 4.0 doesn't support the 'tags' tag. Do you think I'll be safe to remove the 'tags' tags? 

I do not want to 'upgrade' to Zabbix 4.2 because 4.0 is the latest LTS release and hence will be supported longer than 4.2. It would be great to see this template modified to support 4.0 or if a 4.0 template was included too.

**Comment by danboid** [comment]
OK, the 'tags' tags aren't the only issue for 4.0. I removed all of the tags tags and I get this error now:

```
Import failed

    Invalid tag "/zabbix_export/templates/template(1)/discovery_rules/discovery_rule(1)": unexpected tag "lld_macro_paths".
```

**Comment by rvalitov** [comment]
Sorry for a late reply. Thank you very much for your tests with Zabbix 4.0. It seems that the `lld_macro_paths` tag appeared in Zabbix 4.2 and was not available in Zabbix 4.0 ([issue](https://support.zabbix.com/browse/ZBXNEXT-4877), [docs](https://www.zabbix.com/documentation/4.2/manual/discovery/low_level_discovery#discovery_rule_custom_macros)).

As far as I can see, my code does not use the `lld_macro_paths` tag. It was inserted automatically by my Zabbix 4.2 when I exported the template. You can try to safely delete this tag.

I don't have Zabbix 4.0 by hand, but I will install it and try to import the template there.

**Comment by rvalitov** [comment]
easy bot

**Comment by yantsa** [comment]
I removed tags, removed lld_macto_paths but I get: 

Invalid tag "/zabbix_export/templates/template(1)/discovery_rules/discovery_rule(1)": unexpected tag "preprocessing".

**Comment by rvalitov** [comment]
Thank you! It seems I have to install this version of Zabbix myself and run some tests. Give me one week and I will come back with some solution.

**Comment by rvalitov** [comment]
@danboid 
@yantsa 
I made a new PR #15. An new Zabbix template is added there for Zabbix 4.0. Please, check and test it. Your feedback is much appreciated.

**Comment by rvalitov** [comment]
Closed by #15
